### DISCUSSION TITLE
How to throttling the sending rate?

### DISCUSSION THREAD

**Discussion by Pacerino** [haraka/Haraka#3242] [Q&A] [answered]
I've set up haraka to relay incoming mails to Amazon SES. I'm limited by 14 Mails / Second. Is there a way to throttle the rate at which the smtp_forward plugin is forwarding the messages to Amazon SES?

**Comment by msimerson** [comment, marked as answer]
Look at the [limit](https://github.com/haraka/haraka-plugin-limit) plugin. It's a very good starting place.

**Reply by Pacerino** [reply]
I've actually tried to set it up but its not working. Limit + Reds plugins are enabled but im getting an error. `TypeError: this.merge_redis_ini is not a function`.

**Reply by msimerson** [reply]
Do you have the redis plug-in in your `config/plugins` file?

**Reply by Pacerino** [reply]
As mentioned in this issue, it was related to an outdated plugin. But as also mentioned hiredis should be replaced as it is archived since Nov 2019.
https://github.com/haraka/Haraka/issues/3243
